manage/sawtooth_manage/docker.py

```diff
@@ -135,10 +135,10 @@ def start(self, node_config):
         if node_config.genesis:
             command = 'bash -c "sawtooth admin keygen && \
             sawtooth admin genesis && \
-            validator {} -v"'
+            validator {} -v --public-uri tcp://0.0.0.0:8800"'
```

Comment:
0.0.0.0 is not valid for --public-uri. The intent is this is a reachable endpoint - it should be a hostname (preferably) or an ip address for how this docker container is reachable by other peers on the cluster.